src/queries.ts

```diff
@@ -1,18 +1,17 @@
-import { gql } from "graphql.macro";
+import { graphql } from "./common/graphql/types";
```

Comment:
We must use the generated `graphql` function rather than `graphql.macro` here.

With GraphQL codegen & `client-preset`, we can leverage generated `graphql` functions so we don't have to type `useQuery` manually, and we can rely automatically on generated types. To do so, we need to use the generated `graphql` function rather than `graphql.macro`.